### DISCUSSION TITLE
Should we use the linux release on mac ?

### DISCUSSION THREAD

**Discussion by at0mb0y** [bnpr/Malt#317] [Q&A]
I’m trying to setup Malt on Mac Os 12.4 with Blender 3.1.2. 
I’ve try to install various version of malt but when I tick the box I got error and it doesn’t work.
Is it me or it’s normal, BEER is not suppose to work on mac os ?

**Comment by pragma37** [comment]
Malt is not supported in MacOS anymore, you can read about the reasons here: [*(About MacOS)*](https://github.com/bnpr/Malt/discussions/231).
